app/controllers/idv/gpo_controller.rb

```diff
@@ -70,6 +72,17 @@ def resend_requested?
       current_user.gpo_verification_pending_profile?
     end
 
+    # Caveat: If the user succeeds on their final phone attempt, the :proof_address
+    # RateLimiter is reset to 0. But they probably wouldn't be doing verify by mail
+    # if they succeeded on the phone step.
+    def phone_step_attempts
+      RateLimiter.new(user: current_user, rate_limit_type: :proof_address).attempts
+    end
+
+    def first_letter_requested_at
+      current_user.gpo_verification_pending_profile&.gpo_verification_pending_at
```

Comment:
Is there a better value to use for first_letter_requested_at that will be available both for initial letters and later resends? Or is it ok to have this value be null for the initial letter?

Reply:
Could you do something like this?

```ruby
def first_letter_requested_at
  if resend_requested?
    current_user.gpo_verification_pending_profile.gpo_verification_pending_at
  else
    Time.zone.noe
  end
end
```

Reply:
I could, but then the first_letter_requested_at value would be different for that one event than all the other requested and enqueued events for this profile. Is that better than a null value?